### ISSUE TITLE
CodeMirror.on is not a function trying to use tab

### ISSUE THREAD

**Issue by eMerzh** [graphql/graphiql#2263]
hello, i've just update my project, to the lastest version thus trying to use the new tab feature ✨ ..
i've added the tab props and reloaded but once i type in the editor i have:

> TypeError CodeMirror.on is not a function
> Call Stack
>  onHasCompletion/<
>   node_modules/graphiql/esm/utility/onHasCompletion.js:18:20

arround here:

> 
> export default function onHasCompletion(_cm, data, onHintInformationRender) {
>     import('codemirror').then(function (CodeMirror) {
>         var information;
>         var deprecation;
>         CodeMirror.on(data, 'select', function (ctx, el) {
> 

<img width="570" alt="image" src="https://user-images.githubusercontent.com/177003/160917567-522bbd3e-94bf-43eb-9f29-344956a30742.png">

i've clear up a bunch of customisation to finally have a minimal producing , and it looks like this  : 

```jsx
import ReactDOM from 'react-dom'
import GraphiQL from 'graphiql'
import 'graphiql/graphiql.min.css'
import './styles.css'

function graphQLFetcher(graphQLParams) {
	return fetch('/graphql', {
		method: 'post',
		headers: { 'Content-Type': 'application/json' },
		body: JSON.stringify(graphQLParams),
	}).then(response => response.json())
}

const App = ({ fetcher }) => <GraphiQL style={{ height: '100vh' }} headerEditorEnabled tabs fetcher={fetcher} />
ReactDOM.render(<App fetcher={graphQLFetcher} />, document.getElementById('root'))

```

is there something i can do to fix it? 

note: i'm using webpack

**Comment by eMerzh** [comment]
not sure it's related to tab after all 🤔 
but it triggers on complete if it wasn't clear already

**Comment by acao** [comment]
It’s probably related to a recent change where we switched to dynamic imports instead of inline require for importing codemirror (for SSR support). I will try to re-create the bug soon

**Comment by eMerzh** [comment]
ok thanks @acao  🙌 

it seems to be related to this line https://github.com/graphql/graphiql/blob/main/packages/graphiql/src/utility/onHasCompletion.ts#L29  so yeah import might be the issue 🤔

**Comment by acao** [comment]
Fixed in 1.8.3!
